contrib/IECoreMantra/test/IECoreMantra/RendererTest.py

```diff
@@ -196,12 +196,13 @@ def testShaderParameters( self ):
 		self.assertTrue( world )
 		self.assertEquals( world.typeId(), IECore.Group.staticTypeId() )
 		self.assertTrue( world.state() )
-		self.assertEquals( 
-			world.state()[0].attributes[':surface'], 
-			IECore.StringData( 'testshader p2 1.234 p3 "hello" p1 11 p4 1 2 3 p5 1 0 0 ')
+		self.assertItemsEqual(
```

Comment:
I think the order matters here. What was the failure before adding the split() ?